src/helpers/federalEin.ts

```diff
@@ -0,0 +1,19 @@
+import { useTranslation } from 'react-i18next'
+
+export const normalizeEin = (value: string) =>
```

Comment:
This is just a light nit.... maybe we need like a comment/doc above this function to explain what it is and why/how it does what it does. Just reading the chaining of match,slice,filter,join,substring made my head go 🤯

Reply:
Agreed, updated!

Reply:
Thank you! Future developers will be thankful